_benchmark/reference/commands/redline-test.md

```diff
@@ -81,4 +107,9 @@ Use the following options and behaviors to better understand and customize redli
 - `--redline-scale-step`: Specifies the number of clients to unpause in each scaling iteration.
 - `--redline-scaledown-percentage`: Specifies the percentage of clients to pause when an error occurs.
 - `--redline-post-scaledown-sleep`: Specifies the number of seconds the feedback actor waits before initiating a scale-up after scaling down.
-- `--redline-max-clients`: Specifies the maximum number of clients allowed during redline testing. If unset, OpenSearch Benchmark defaults to the number of clients defined in the test procedure.
\ No newline at end of file
+- `--redline-max-clients`: Specifies the maximum number of clients allowed during redline testing. If unset, OpenSearch Benchmark defaults to the number of clients defined in the test procedure.
+
+### For CPU-based Feedback
+- `--redline-cpu-max-usage`: (Required) Max allowed CPU load (%) per node before triggering a scale-down.
+- `--redline-cpu-window-seconds`: Duration (in seconds) over which to average CPU usage per node (default: 30).
+- `--redline-cpu-check-interval`: Interval (in seconds) between CPU usage checks (default: 30).
```

Comment:
Should we just mark these as optional and remove the `Optional Tuning Flags` section above as they're redundant?